cyrasterize/c_opengl_debug.pyx

```diff
@@ -14,15 +13,15 @@ class bcolors:
     END = '\x1b[0m'
 
 def printWarn(string, color=bcolors.YELLOW):
-    log.debug(color + string + bcolors.END)
+    log.debug(color + string + color)
 
 cdef print_error():
     code = cgl.glGetError()
     cdef char* c_char = <char*> gluErrorString(code)
     cdef bytes pyString = c_char
 
     if code:
-        log.error(" *** Error *** [{}] {}".format(pyString, code), color=bcolors.RED)
+        log.error(" *** Error *** [{}] {}".format(pyString, code) + bcolors.RED)
```

Comment:
how come we sandwich the string with color above and only append with a color here?